package.json

```diff
@@ -1,213 +1,216 @@
 {
-  "name": "preact",
```

Comment:
I think your liner is doing something odd here, can merge after we fix this

Reply:
@JoviDeCroock You're right! I'm getting mad at VS Code. It kept auto-formatting spaces to tabs, but I have fixed it now! 👍